addons/mail/models/mail_link_preview.py

```diff
@@ -16,6 +16,7 @@ class LinkPreview(models.Model):
     _description = "Store link preview data"
 
     message_id = fields.Many2one('mail.message', string='Message', index=True, ondelete='cascade', required=True)
+    displayed = fields.Boolean("LinkPreview is displayed", default=True)
```

Comment:
It's better to add new fields with a default empty so we don't need to make an upgrade script to add the value. (inverse the meaning of the field to have false)